.github/workflows/integration-test.yml

```diff
@@ -18,76 +18,76 @@ jobs:
         with:
           java-version: 14
 
-      - name: Checkout Kibana
+      - name: Checkout OpenSearch Dashboard
         uses: actions/checkout@v2
         with:
-          path: kibana
-          repository: elastic/kibana
+          path: OpenSearch-Dashboards
+          repository: opensearch-project/OpenSearch-Dashboards
           fetch-depth: 0
       
       - name: Create plugins dir
         run: |
-          cd ./kibana
+          cd ./OpenSearch-Dashboards
           mkdir -p plugins
       
-      - name: Checkout security kibana plugin
+      - name: Checkout OpenSearch Dashboard Security plugin
         uses: actions/checkout@v2
         with:
-          path: kibana/plugins/opendistro_security
+          path: OpenSearch-Dashboards/plugins/security-dashboards-plugin
           ref: ${{ github.ref }}
       
-      - name: Get Kibana version
-        id: kbn_version
+      - name: Get OpenSearch Dashboards version
+        id: osd_version
         run: |
-          echo "::set-output name=kbn_version::$(jq -r '.kibana.version' ./kibana/plugins/opendistro_security/package.json)"
+          echo "::set-output name=osd_version::$(jq -r '.opensearchDashboards.version' ./OpenSearch-Dashboards/plugins/security-dashboards-plugin/package.json)"
       
-      - name: Get Opendistro version
+      - name: Get OpenSearch version
         id: opendistro_version
         run: |
-          echo "::set-output name=opendistro_version::$(jq -r '.version' ./kibana/plugins/opendistro_security/package.json | sed 's/[0-9]*$/0/')"
+          echo "::set-output name=opendistro_version::$(jq -r '.version' ./OpenSearch-Dashboards/plugins/security-dashboards-plugin/package.json | sed 's/[0-9]*$/0/')"
 
-      - name: Run elasticsearch with plugin
+      - name: Run elasticsearch with plugin (es 7.10.2, odfe 1.13)
         run: |
-          wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-${{ steps.kbn_version.outputs.kbn_version }}-linux-x86_64.tar.gz
-          tar -xzf elasticsearch-oss-${{ steps.kbn_version.outputs.kbn_version }}-linux-x86_64.tar.gz
-          cd elasticsearch-${{ steps.kbn_version.outputs.kbn_version }}/
-          bin/elasticsearch-plugin install -b https://d3g5vo6xdbdb9a.cloudfront.net/downloads/elasticsearch-plugins/opendistro-security/opendistro-security-${{ steps.opendistro_version.outputs.opendistro_version }}.zip
+          wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-7.10.2-linux-x86_64.tar.gz
```

Comment:
All of these will need to be updated when OpenSearch docker images are available?